assets/src/edit-story/elements/index.js

```diff
@@ -34,7 +34,10 @@ import * as videoElement from './video';
 
 export const createNewElement = (type, attributes = {}) => {
   const element = elementTypes.find((el) => el.type === type);
-  const defaultAttributes = element ? element.defaultAttributes : {};
+  if (!element) {
+    throw new Error(`Unknown element type: ${type}`);
+  }
+  const { defaultAttributes } = element;
```

Comment:
I updated this to extend test coverage of `createNewElement` to 100%. If there's no element type found, we really shouldn't proceed here - that would cause all sorts of weirdness.